jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -281,6 +285,11 @@ defaults
   <%- if p("ha_proxy.backend_prefer_local_az") -%>
     option allbackups
   <%- end -%>
+  <%- if p("ha_proxy.enable_redispatch") -%>
+    option redispatch
+  <%- end -%>
+    retries <%= p("ha_proxy.retries") %>
```

Comment:
Maybe move this into the if as well? I'd prefer if we keep the configuration as small as possible.

Reply:
they are two distinct properties. you can have retries without redispatch. but I can add it only if the value is larger than 0

Reply:
Ah, got it. In that case I don't have a preference.